js/packages/mobile-wallet-adapter-protocol-web3js/src/transact.ts

```diff
@@ -16,8 +16,8 @@ import { fromUint8Array, toUint8Array } from './base64Utils';
 
 interface Web3SignAndSendTransactionsAPI {
     signAndSendTransactions(params: {
-        connection: Connection;
-        fee_payer?: PublicKey;
+        commitment: Finality;
+        preflight_commitment: Finality;
```

Comment:
We could presumably use whichever is the lower threshold of these to get the blockhash

Reply:
The logic being, the blockhash may fail to be found if the `preflightCommitment` is `finalized` (the default) and the blockhash we're basing it on is only `confirmed` (seemingly the cause of https://github.com/brave/brave-browser/issues/24345)

Reply:
The aim of this PR is to leave that work up to whoever calls into `mobile-wallet-adapter-protocol-web3js`.

Reply:
Which is to say I'll add this logic to the wallet adapter plugin.

Reply:
Perhaps I'm missing something, but with the latest changes (which look good) moving things to WAM, are these args used anymore?

Reply:
Ah, are these just passed through with `...rest`?